juri/weather_widget/script.js

```diff
@@ -0,0 +1,77 @@
+
+const form = document.querySelector(".top-banner form");
+const input = document.querySelector(".top-banner input");
+const msg = document.querySelector(".top-banner .msg");
+const list = document.querySelector(".ajax-section .cities");
+const apiKey = "bbc6194de3e2ba3c0decc02f56ce65e2";
+
+form.addEventListener("submit", e => {
```

Comment:
This is a so called code smell "Long method": https://refactoring.guru/smells/long-method.
Consider split this one huge method into small one single purpose functions.